go/exorelay/exorelay_test.go

```diff
@@ -126,6 +164,56 @@ func FeatureContext(s *godog.Suite) {
 		}
 		return nil
 	})
+
+	s.Step(`^I register a handler for the "([^"]*)" message$`, func(messageName string) error {
+		mapping := map[string]func(map[string]interface{}){}
+		mapping[messageName] = func(payload map[string]interface{}) {
+			handlerCalled = true
+			savedPayload = payload
+		}
+		exoInstance.BindMessageHandlers(mapping)
```

Comment:
Here is an example of binding message handlers. You do them all at once, supplying a map from the message name to a function. This is similar to how @alexdavid handled a similar problem on pairfoce.